### PR TITLE
Changed routes

### LINKED ISSUE #774: Consistency in View Component paths
**Is your feature request related to a problem? Please describe.**
Currently, Accounting & Self service sections of the web app use `view/:id` as the path to view components but other sections use `:id` path to view components.

**Describe the solution you'd like**
I would like the view component path to be consistent throughout the web app, i.e., change the view components paths of Accounting & Self service sections.

**Additional context**

### PR DESCRIPTION
**Pull request by Abhirup-99**
## Description
Changing route names to bring about parity.

## Related issues and discussion
Fixes #774 

## Checklist
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] If you have multiple commits please combine them into one commit by squashing them.

- [x] Read and understood the contribution guidelines at `web-app/.github/CONTRIBUTING.md`.

### CODE REVIEW

**Review by karantakalkar — CHANGES_REQUESTED** [review]
Change all `view/:id` routes similarly in this file, also please follow commit message guidelines.

**Comment by Abhirup-99** [comment]
@karantakalkar please review. 
**Note**: I have changed position of 1 route to ensure it doesn't bother any other routes.

**Review by karantakalkar — CHANGES_REQUESTED** [review]
you will need to change routing of all buttons where these routes are accessible from `view/:id` to `:id`, test data can be found on demo.mifos.io, you can change server in environment.ts. 
Please test routing of all changed pages before requesting review.

**Comment by Abhirup-99** [comment]
@karantakalkar do talk a look now. It seems ready for review.

**Comment by Abhirup-99** [comment]
@karantakalkar can you take a look into this? I think it's ready for review now.

File: src/app/accounting/chart-of-accounts/create-gl-account/create-gl-account.component.ts [hunk 1]
```diff
@@ -118,7 +118,7 @@ export class CreateGlAccountComponent implements OnInit {
    */
   submit() {
     this.accountingService.createGlAccount(this.glAccountForm.value).subscribe((response: any) => {
-      this.router.navigate(['../view', response.resourceId], { relativeTo: this.route });
+      this.router.navigate([`${response.resourceId}`], { relativeTo: this.route });
```
Comment (karantakalkar) [inline src/app/accounting/chart-of-accounts/create-gl-account/create-gl-account.component.ts:121]:
```suggestion
      this.router.navigate([response.resourceId], { relativeTo: this.route });
```

**Review by karantakalkar — CHANGES_REQUESTED** [review]
are you sure you have checked all buttons/mat table rows for routing changes? and tested each route change?

**Comment by Abhirup-99** [comment]
@karantakalkar every route has been checked thoroughly.

**Comment by Abhirup-99** [comment]
@karantakalkar your changes are incoporated.

**Comment by Abhirup-99** [comment]
@karantakalkar can you review this pr, it's sitting here for a long time.

**Pull request closed without merging**